hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/controller/NetworkController.java

```diff
@@ -88,10 +88,10 @@ private NodeAddress toNodeAddress(AddressBookEntry addressBookEntry) {
         for (var s : addressBookEntry.getServiceEndpoints()) {
             try {
                 var ipAddressV4 = InetAddress.getByName(s.getIpAddressV4()).getAddress();
-                var serviceEndpoint = ServiceEndpoint.newBuilder()
-                        .setIpAddressV4(ProtoUtil.toByteString(ipAddressV4))
-                        .setPort(s.getPort())
-                        .build();
+                var serviceEndpoint = ServiceEndpoint.newBuilder().setIpAddressV4(ProtoUtil.toByteString(ipAddressV4));
+                serviceEndpoint.setDomainName(s.getDomainName());
+                serviceEndpoint.setPort(s.getPort());
+                serviceEndpoint.build();
```

Comment:
Weird mix of builder and setters. Also sort.
```suggestion
                var serviceEndpoint = ServiceEndpoint.newBuilder()
                    .setDomainName(s.getDomainName())
                    .setIpAddressV4(ProtoUtil.toByteString(ipAddressV4))
                    .setPort(s.getPort())
                    .build();
```